test/test.js

```diff
@@ -5,24 +5,24 @@ var MetaInspector = require('../index'),
 
 require('./fixtures/fixtures');
 
-describe('metainspector', function(){
```

Comment:
Let's keep the changes to the paren spacing out of this PR especially since it doesn't match the rest of the codebase.